cmd/crc/cmd/stop.go

```diff
@@ -54,7 +55,7 @@ func runStop(arguments []string) {
 				errors.Exit(0)
 			}
 			var userInput string
-			output.OutF("Do you want to force power off [y/n]: ")
+			output.OutF("Do you want to force power off [y/N]: ")
 			fmt.Scan(&userInput)
 			if strings.ToLower(userInput) == "y" {
```

Comment:
this would be removed in that case.

Reply:
This will be automatically handled by PromptForUserYesNo ;)